src/schema/schema_create.c

```diff
@@ -8,6 +8,22 @@
 
 #include "wt_internal.h"
 
+/* Holds metadata entry name and the associated config string. */
+typedef struct {
+    char *name;
+    char *config;
+} WT_IMPORT_ENTRY;
+
+/* Array of metadata entries used in the import from metadata file process. */
```

Comment:
The word 'process' is confusing here (makes me think of a Unix process). Maybe:
Array of metadata entries used when importing from a metadata file.

Reply:
Done. Thank you for the suggestion.